src/lib/serviceRoutines/putAllEntitiesWithTypeAndId.cpp

```diff
@@ -34,12 +34,26 @@
 #include "convenience/UpdateContextElementResponse.h"
 #include "convenienceMap/mapPutIndividualContextEntity.h"
 #include "serviceRoutines/putAllEntitiesWithTypeAndId.h"
+#include "serviceRoutines/postUpdateContext.h"
 
 
 
 /* ****************************************************************************
 *
 * putAllEntitiesWithTypeAndId - 
+*
+* PUT /v1/contextEntities/type/{entity::type}/id/{entity::id}
+*
+* Payload In:  UpdateContextElementRequest
+* Payload Out: UpdateContextElementResponse
+*
+* URI parameters:
+*   - attributesFormat=object
+*
```

Comment:
Probably you have already though on that :) but the .test should be also updated to include them.